backend/isoko-backend/src/handlers/user/get-user-object.js

```diff
@@ -1,21 +1,53 @@
+const dynamodb = require('aws-sdk/clients/dynamodb');
+const docClient = new dynamodb.DocumentClient();
+const _ = require('lodash');
+const { USER_TABLE } = require('../../constants');
 /**
  * HTTP get method to geta user object based off of the userSub specified in the request.
-
  */
 exports.getUserObjectHandler = async (event) => {
    if (event.httpMethod !== 'GET') {
-      throw new Error(
-         `getUserObject only accept GET method, you tried: ${event.httpMethod}`
-      );
+      return {
+         statusCode: 400,
+         body: { error: `getUserObject only accept GET method, you tried: ${event.httpMethod}` },
+      };
    }
 
    console.info('received:', event);
+   const pk = _.get(event.pathParameters, 'pk', null);
+
+   if (pk == null) {
+      return {
+         statusCode: 400,
+         body: { error: `Missing query parameter 'pk'. Request URL format: GET/user/{pk}` },
+      };
+   }
 
-   const response = {
-      statusCode: 200,
-      body: { ...JSON.parse(event.body), requestParams: event.pathParameters },
+   const params = {
+      TableName: USER_TABLE,
+      Key: {
+         pk: pk,
+      },
    };
 
+   let response;
+
+   try {
+      const dynamoResult = await docClient.get(params).promise();
+
+      let getResults = dynamoResult.Item;
+
+      response = {
+         statusCode: 200,
+         body: { results: getResults },
+      };
```

Comment:
This probably won't work because we're missing the CORS headers and the application type json stuff. Try deploying to your local account and testing the endpoint. If it doesn't work then you'll probably need to do something similar to this. Also just return the results directly in the body not under a results property

```
      response = {
         statusCode: 200,
         body: JSON.stringify(businessDetails),
         headers: {
            'content-type': 'json',
            'access-control-allow-origin': '*',
         },
      };
```